includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php

```diff
@@ -268,6 +268,15 @@ public function getData() {
 						'href' => 'https://www.dndbeyond.com/',
 						'tracking_label' => 'explore.dnd-beyond',
 					],
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'text',
+							'value' => 'Cortex RPG'
+						],
+						'href' => 'https://www.cortexrpg.com/',
```

Comment:
You can exclude the `www.` -- it always redirects to cortexrpg.com anyway.

Reply:
Any harm in leaving it?